src/app/core/facades/checkout.facade.ts

```diff
@@ -68,13 +78,7 @@ export class CheckoutFacade {
   // BASKET
 
   basket$ = this.store.pipe(select(getCurrentBasket));
-  basketChange$ = this.store.pipe(
-    select(getBasketLastTimeProductAdded),
-    whenTruthy(),
-    distinctUntilChanged(),
-    switchMap(() => this.basketLoading$.pipe(debounceTime(500), whenFalsy())),
-    shareReplay(1)
```

Comment:
`shareReplay` is not a solution here, because the header is re-initialized via `AppFacade.headerType$` when leaving the checkout. We need a "global" `Observable` that only emits when new changes happen. Using only NgRx is also not a good solution as all subscriptions emit the last value and working with `skip(1)` on subscribe is masking quite a lot (also not working btw). The only probable solution might really be local state in the facade...

cc @shauke @Eisie96